MapboxNavigation/ImageDownloader.swift

```diff
@@ -1,7 +1,59 @@
 import Foundation
+import MapboxDirections
 
 typealias ImageDownloadCompletionBlock = (UIImage?, Data?, Error?) -> Void
 
+/// The user agent string for any HTTP requests performed directly within this library.
+let userAgent: String = {
```

Comment:
There are a couple other spots in MapboxCoreNavigation that kick off URLSession data tasks. Perhaps they should also use this user agent string for consistency. We could move this lazily computed constant to an extension on URLSession.

https://github.com/mapbox/mapbox-navigation-ios/blob/71a5de2c77a7945e2fde984fbd34e3198012dc89/MapboxCoreNavigation/LegacyRouteController.swift#L392 https://github.com/mapbox/mapbox-navigation-ios/blob/71a5de2c77a7945e2fde984fbd34e3198012dc89/MapboxCoreNavigation/NavigationService.swift#L535